zomes/rea_economic_resource/storage/src/lib.rs

```diff
@@ -236,6 +244,12 @@ impl Updateable<EventCreateRequest> for EntryData {
             } else { self.current_location.to_owned() },
             contained_in: self.contained_in.to_owned(),
             note: self.note.to_owned(),
+            primary_accountable: if e.to_resource_inventoried_as.to_owned().is_some() && e.get_action() == "raise" {
```

Comment:
```suggestion
            // NOTE: this could be "dangerous" in the sense that if not validated properly, this ability to update via events could be abused by third party agents transferring rights and 'ownership' to themselves, from resources currently controlled/owned/stewarded by other agents
            primary_accountable: if e.to_resource_inventoried_as.to_owned().is_some() && (e.get_action() == "transfer" || e.get_action() == "transfer_all_rights") {
```